TrainingProject/src/main/java/com/example/trainningspringproject/entity/Machine.java

```diff
@@ -0,0 +1,60 @@
+package com.example.trainningspringproject.entity;
+
+import jakarta.persistence.Entity;
+import jakarta.persistence.GeneratedValue;
+import jakarta.persistence.GenerationType;
+import jakarta.persistence.Id;
+
+@Entity
+
+public class Machine {
+    @Id
+    @GeneratedValue(strategy = GenerationType.IDENTITY)
+    private int Id;
+    private String Name;
+    private String Location;
+    private int user;
```

Comment:
we agreed on creating a table of users and her you should use a userId connection to that table